neon_core/util/skill_utils.py

```diff
@@ -75,14 +75,12 @@ def get_neon_skills_data(skill_meta_repository: str =
     return skills_data
 
 
-def _write_pip_constraints_to_file(output_file: str = None):
+def _write_pip_constraints_to_file(output_file):
```

Comment:
There doesn't seem to be any error handling in the case that the parameter is falsey. There wasn't before either, but it may be worth adding in this PR